src/main/java/com/snowflake/snowpark_java/Functions.java

```diff
@@ -4504,11 +4504,144 @@ public static Column from_unixtime(Column ut, String f) {
    * [Row(SEQ8(0)=0),Row(SEQ8(0)=1), Row(SEQ8(0)=2)]
    * }</pre>
    *
+   * @return A sequence of monotonically increasing integers, with wrap-around * which happens after
+   *     largest representable integer of integer width 8 byte.
    * @since 1.15.0
    */
   public static Column monotonically_increasing_id() {
     return new Column(com.snowflake.snowpark.functions.monotonically_increasing_id());
   }
+  /**
+   * Returns number of months between dates `start` and `end`.
+   *
+   * <p>A whole number is returned if both inputs have the same day of month or both are the last
+   * day of their respective months. Otherwise, the difference is calculated assuming 31 days per
+   * month.
+   *
+   * <p>For example:
+   *
+   * <pre>{@code
+   * {{{
+   * months_between("2017-11-14", "2017-07-14")  // returns 4.0
+   * months_between("2017-01-01", "2017-01-10")  // returns 0.29032258
+   * months_between("2017-06-01", "2017-06-16 12:00:00")  // returns -0.5
+   * }}}
+   * }</pre>
+   *
+   * @param end A date, timestamp or string. If a string, the data must be in a format that can be
+   *     cast to a timestamp, such as `yyyy-MM-dd` or `yyyy-MM-dd HH:mm:ss.SSSS`
+   * @param start A date, timestamp or string. If a string, the data must be in a format that can
+   *     cast to a timestamp, such as `yyyy-MM-dd` or `yyyy-MM-dd HH:mm:ss.SSSS`
+   * @return A double, or null if either `end` or `start` were strings that could not be cast to a
+   *     timestamp. Negative if `end` is before `start`
+   * @since 1.15.0
+   */
+  public static Column months_between(Column end, Column start) {
```

Comment:
consider adding an overload that receives column names (not column objects) for end and start

Reply:
Fixed it.Can you please check now.